readthedocs/doc_builder/environments.py

```diff
@@ -14,7 +14,7 @@
 from docker.errors import APIError as DockerAPIError
 from docker.errors import DockerException
 from docker.errors import NotFound as DockerNotFoundError
-from requests.exceptions import ConnectionError, ReadTimeout
+from requests.exceptions import ConnectionError, ReadTimeout  # noqa
```

Comment:
Why is the noqa needed? Is there a more explicit way to note what we're actually disabling?

Reply:
`ConnectionError` is a built-in exception and pylint says it can't be overwritten 🤷🏼

Reply:
Can we import it another way?

`import ConnectionError as RequestConnectionError` or similar